scss/atoms/typography/_typography.scss

```diff
@@ -124,5 +132,5 @@ code {
 	background: #f2f2f2; // TODO replace with color variable
 	font-size: em( 16px );
 	padding: em( 4px, 16px ) em( 8px, 16px );
-	border-radius: 3px;
+	border-radius: 3px; // TODO replace with variable
```

Comment:
makes sense.. hence i added a todo to check if you agree there should be a variable..I'll update today with the changes